src/main/java/org/umssdiplo/automationv01/core/managepage/organizationalstructuremenu/OrganizationalStructureMenu.java

```diff
@@ -8,19 +8,25 @@
 import org.umssdiplo.automationv01.core.utils.CommonEvents;
 
 public class OrganizationalStructureMenu extends BasePage {
+
+    @FindBy(xpath = "//span[text() = 'Estructura Organizacional']")
+    private WebElement employeeMenu;
+
     @FindBy(id = "sub-menu-positions")
     private WebElement positionSubMenu;
 
-    @FindBy(id = "sub-menu-employees")
+    @FindBy(xpath = "//span[text() = 'Empleados']")
```

Comment:
Why are you changing this to xpath? this menu already have their id. Let's use the one we already have